Data/light-dark.txt

```diff
@@ -1,493 +0,0 @@
-bulbasaur 0.61604
-ivysaur 0.462721
-venusaur 0.5451
-charmander 0.724984
-charmeleon 0.491232
-charizard 0.601881
-squirtle 0.708672
-wartortle 0.748101
-blastoise 0.64938
-caterpie 0.718227
-metapod 0.621901
-butterfree 0.630765
-weedle 0.638418
-kakuna 0.773358
-beedrill 0.802686
-pidgey 0.460133
-pidgeotto 0.906494
-pidgeot 0.887968
-rattata 0.478938
-raticate 0.640308
-spearow 0.459023
-fearow 0.488446
-ekans 0.463613
-arbok 0.589331
-pikachu 0.851867
-raichu 0.751316
-sandshrew 0.77978
-sandslash 0.730616
-nidoran 0.759855
-nidorina 0.681913
-nidoqueen 0.633158
-nidoran 0.414212
```

Comment:
as in not a literally, bus this will definitely cause issue what happens on `pokemon nidoran`